src/fvdb/GaussianSplat3d.cpp

```diff
@@ -1050,6 +1052,27 @@ GaussianSplat3d::renderImagesAndDepths(const torch::Tensor &worldToCameraMatrice
         state, settings.tileSize, settings.imageWidth, settings.imageHeight, 0, 0, backgrounds);
 }
 
+std::tuple<torch::Tensor, torch::Tensor>
+GaussianSplat3d::relocateGaussians(const torch::Tensor &logScales,
+                                   const torch::Tensor &logitOpacities,
+                                   const torch::Tensor &ratios,
+                                   const torch::Tensor &binomialCoeffs,
+                                   const int nMax,
+                                   const float minOpacity) {
+    return FVDB_DISPATCH_KERNEL_DEVICE(logScales.device(), [&]() {
```

Comment:
This can be `FVDB_DISPATCH_KERNEL` because the `kPrivateUse1` specialization in populated.